src/twisted/python/failure.py

```diff
@@ -90,30 +90,41 @@ class NoCurrentExceptionError(Exception):
 
 
 
-class _Traceback(object):
+def _Traceback(frames):
```

Comment:
I'm a little worried about switching this to a function, but it's a private API and it doesn't seem to be called anywhere outside of `twisted.python.failure`.